### PR TITLE
feat: metrics for errors & backwards compat metric names with chain-mon

### PR DESCRIPTION
**Pull request by hamdiallam**
missing the same error metrics present in chain-mon.

Makes sure the namespaces matches the configuration in chain-mon

### CODE REVIEW

File: op-monitorism/multisig/monitor.go [hunk 1]
```diff
@@ -122,7 +129,7 @@ func (m *Monitor) checkOptimismPortal(ctx context.Context) {
 		pausedMetric = 1
 	}
 
-	m.paused.WithLabelValues(m.optimismPortalAddress.String(), m.nickname).Set(float64(pausedMetric))
+	m.pausedState.WithLabelValues(m.optimismPortalAddress.String(), m.nickname).Set(float64(pausedMetric))
```
Comment (Ethnical) [inline op-monitorism/multisig/monitor.go:132]: The line is using the `optimismPortalAddress` that is correct here.
```go
m.pausedState.WithLabelValues(m.optimismPortalAddress.String(), m.nickname).Set(float64(pausedMetric))
```

However, the **L192** (that is outside the current diff) is also using the `optimismPortalAddress` which is not correct there: 
```go
m.latestPresignedPauseNonce.WithLabelValues(m.optimismPortalAddress.String(), m.nickname).Set(float64(latestPresignedNonce))
```
The metrics should be the `m.safeAddress` I guess.

File: op-monitorism/multisig/monitor.go [hunk 2]
```diff
@@ -170,6 +180,7 @@ func (m *Monitor) checkPresignedNonce(ctx context.Context) {
 			nonce, err := strconv.ParseInt(item.Title[6:len(item.Title)-5], 10, 64)
 			if err != nil {
 				m.log.Error("failed to parse nonce from item title", "title", item.Title)
+				m.unexpectedRpcErrors.WithLabelValues("1pass", "title").Inc()
```
Comment (Ethnical) [inline op-monitorism/multisig/monitor.go:183]: NITS: Usage of magic numbers for (`6`, `5`, `10`) should we remove them to improve readibility? 
```go
180                 nonce, err := strconv.ParseInt(item.Title[6:len(item.Title)-5], 10, 64) // Should we remove the magic number maybe overkill but improve readability in the long run. 
```

**Review by Ethnical — CHANGES_REQUESTED** [review]
If we could set the correct address for the `safeAddress`.

File: op-monitorism/multisig/monitor.go (hunk 1 above)
Comment (hamdiallam) [reply op-monitorism/multisig/monitor.go:132]: ahh good catch!

File: op-monitorism/multisig/monitor.go (hunk 2 above)
Comment (hamdiallam) [reply op-monitorism/multisig/monitor.go:183]: yea sounds good i'll create some variables

**Review by Ethnical — APPROVED** [review]
Perfect!

**Pull request merged**